hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/bootstrap/SparkBootstrapCommitActionExecutor.java

```diff
@@ -284,7 +284,7 @@ protected void commit(Option<Map<String, String>> extraMetadata, HoodieWriteMeta
   protected BaseSparkCommitActionExecutor<T> getBulkInsertActionExecutor(JavaRDD<HoodieRecord> inputRecordsRDD) {
     return new SparkBulkInsertCommitActionExecutor((HoodieSparkEngineContext) context, new HoodieWriteConfig.Builder().withProps(config.getProps())
         .withSchema(bootstrapSchema).build(), table, HoodieTimeline.FULL_BOOTSTRAP_INSTANT_TS,
-        inputRecordsRDD, extraMetadata);
+        inputRecordsRDD, Option.empty(), extraMetadata);
```

Comment:
this results in error  "java.util.HashMap cannot be cast to org.apache.hudi.table.BulkInsertPartitioner"